patches/tModLoader/Terraria/Localization/Content/ru-RU/tModLoader.json

```
@@ -3,352 +3,353 @@
-		"MonthlyFreezeNotification": "Ежемесячная заморозка альфа-версии началась. Напоминаем, с 21 числа каждого месяца начинается период времени, когда разработчикам модов следует обновить свои моды до версии 1.4-preview в следствии с изменениями кода и API которые были сделаны за месяц. Текущая версия 1.4-preview станет новой 1.4-stable 1-го числа следующего месяца.\n\nНе обновлённые моды возможно перестанут корректно работать 1-го числа.\n\nПрисоединяйтесь к Discord-серверу tModLoader что бы узнать больше об изменениях этого месяца и заметках об обновлении модов. Нажмите \"Больше информации\" что бы узнать больше о цикле релиза альфа-версии.",
-		"FirstLaunchWelcomeMessage": "Добро пожаловать в tModLoader.\n\nУстановка была произведена успешно. Используйте меню 'Браузер Модов' в меню мастерской для того чтобы находить и устанавливать понравившиеся вам моды. Используйте меню 'Управление Модами' для выбора активных модов. После переключения модов не забывайте кликнуть на 'Перезагрузить моды', чтобы изменения вступили в силу.",
-		"WhatsNewMessage": "Далее идёт список изменений с последнего запуска tModLoader Alpha. Возможно, моды которые относятся к этим изменениям должны быть пересобраны их автором. Больше информации на GitHub:\n",
-		"ShowNewUpdatedModsInfoMessage": "Следующие моды были добавлены или обновлены с предыдущего запуска (это сообщение можно отключить в настройках):\n",
-		"ShowNewUpdatedModsInfoMessageNewMods": "\nНовые моды:",
-		"ShowNewUpdatedModsInfoMessageUpdatedMods": "\nОбновлённые моды:",
+		"WelcomeMessageBeta": "Добро пожаловать в альфа-версию tModLoader 1.4!\n\nВы установили нестабильную версию tModLoader, которая может содержать баги, ломающие игру.\n\nИспользуйте меню «Мастерская» для доступа к функционалу модов tModLoader.\n\nПрисоединитесь к Discord серверу tModLoader, чтобы узнать о тестировании модах и/или инструментах, доступных в tModLoader в рамках обновления 1.4.",
+		"MonthlyFreezeNotification": "Ежемесячная заморозка альфа-версии началась. Напоминаем, что 21 числа каждого месяца наступает период, когда разработчикам модов следует обновить свои моды до версии 1.4-preview в следствии с изменениями кода и API, произведёнными за прошедший месяц. Текущая 1.4-preview станет новой 1.4-stable с 1-го числа следующего месяца.\n\nНесоблюдение этого правила означает, что созданные вами моды, вероятнее всего, не будут запускаться, начиная с 1-го числа следующего месяца.\n\nПрисоединитесь к Discord серверу tModLoader, чтобы узнать об изменениях этого месяца и ознакомиться с заметками по переносу модов на новую версию. Нажмите на «Больше информации», чтобы узнать о постоянном цикле релизов альфа-версии.",
```

Changes 'alpha' to 'preview'. 'Preview' is annoying to translate correctly, but an assumption that developers would know at least a bit of English is fair to make.

```suggestion
		"MonthlyFreezeNotification": "Ежемесячная заморозка Preview-версии началась. Напоминаем, что 21 числа каждого месяца наступает период, когда разработчикам модов следует обновить свои моды до версии 1.4-preview в следствии с изменениями кода и API, произведёнными за прошедший месяц. Текущая 1.4-preview станет новой 1.4-stable с 1-го числа следующего месяца.\n\nНесоблюдение этого правила означает, что созданные вами моды, вероятнее всего, не будут запускаться, начиная с 1-го числа следующего месяца.\n\nПрисоединитесь к Discord серверу tModLoader, чтобы узнать об изменениях этого месяца и ознакомиться с заметками по переносу модов на новую версию. Нажмите на «Больше информации», чтобы узнать о постоянном цикле релизов Preview-версий.",
```